frontend/src/components/EditProfile/EditProfile.scss

```diff
@@ -0,0 +1,125 @@
+.edit-profile-form label {
```

Comment:
can I ask you to give your encompassing div a className and then wrap all your associate styles with `.chosenClassName{}`?